--- ostree-2017.1/libglnx/glnx-fdio.h~	2016-10-27 17:30:11.000000000 +0000
+++ ostree-2017.1/libglnx/glnx-fdio.h	2017-10-26 14:54:58.086941259 +0000
@@ -33,7 +33,8 @@
  * undefine basename() since libgen.h defines it as a macro to the XDG
  * version which is really broken. */
 #include <libgen.h>
 #undef basename
+extern char *basename (const char *__filename) __THROW __nonnull ((1));
 
 G_BEGIN_DECLS
 
